projects/plugins/jetpack/extensions/blocks/related-posts/edit.js

```diff
@@ -195,6 +202,21 @@ export class RelatedPostsEdit extends Component {
 
 				<div className={ className } id={ `related-posts-${ instanceId }` }>
 					<div className={ previewClassName } data-layout={ postLayout }>
+						{ displayHeadline && (
+							<InnerBlocks
+								lock={ { remove: false } }
+								template={ [
+									[
+										'core/heading',
+										{
+											className: `jp-related-posts-i2__headline`,
+											level: 3,
+											placeholder: __( 'Related Posts Title', 'jetpack' ),
```

Comment:
Yeah, maybe not even using a placeholder but adding the `Related` title immediately would work. Since it's Gutenberg, I would expect users to try to interact with the block as they would with any other block.